substrate-archive/src/actors/workers/metadata.rs

```diff
@@ -80,7 +80,7 @@ impl<B: BlockT + Unpin> MetadataActor<B> {
 		for b in versions.iter() {
 			self.meta_checker(b.spec, b.inner.block.hash()).await?;
 		}
-		self.addr.send(blks.into()).await?;
+		self.addr.send(blks.into()).await?.await;
```

Comment:
dumb question: is this actix related which is some kind of wrapper type that implements future which return an inner future?

It's really easy miss calling the inner future FWIW.

Reply:
@insipx @niklasad1  the same bug

https://github.com/paritytech/substrate-archive/blob/4ecb1b4bd02a3351041c9fb5f18d14509e82b6cb/substrate-archive/src/actors/workers/storage_aggregator.rs#L95

Reply:
Yeah, @niklasad1 It's a module I made to run multiple actors at once. Planning to fix it soon, described in issue #184 though